changelog.go

```diff
@@ -104,3 +98,12 @@ func CopyChangelogs(changelogs []Changelog) []Changelog {
 
 	return copy
 }
+
+func ValidateKind(v Kind) bool {
```

Comment:
Can't we have `NewKind` that takes `string` and do validation at the time of creation? That way it would be a single call instead of two.

Reply:
Okay, that's what we have in the first place in this PR 😆
see https://github.com/giantswarm/versionbundle/pull/66/commits/c348ede24f6040e008a301b235caa3865885bfbd 

If agree, then I will rollback to the previous state.

Reply:
Done, same function but different name as `NewKind`